File: server/src/main/java/pl/bk/pizza/store/application/dto/order/AddressDTO.java
package pl.bk.pizza.store.application.dto.order;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class AddressDTO
{
    private String city;
    private String street;
    private String streetNumber;
    private String postCode;
}

File: client/pizza-store/src/app/order/addressDTO.ts
export class AddressDTO {
  city: String;
  street: String;
  streetNumber: String;
  postCode: String;

  constructor(data:any) {
    this.city = data.city;
    this.street = data.street;
    this.streetNumber = data.streetNumber;
    this.postCode = data.postCode;
  }

}
